android/src/main/java/com/google/samples/apps/iosched/ui/CurrentSessionActivity.java

```diff
@@ -0,0 +1,1374 @@
+/*
```

Comment:
why do you need to copy it fully? can't you just call and or extend existing functionality?